configure

```diff
@@ -6481,10 +6493,13 @@ EOF
 for engine in $_sorted_engines; do
 	if test "`get_engine_sub $engine`" = "no" ; then
 		j=`echo $engine | tr '[:lower:]' '[:upper:]'`
-		cat >> engines/plugins_table.h << EOF
+		cat >> engines/plugins_table.h.new << EOF
 #if PLUGIN_ENABLED_STATIC($j)
 LINK_PLUGIN($j)
 #endif
 EOF
 	fi
 done
+copy_if_changed engines/plugins_table.h.new engines/plugins_table.h
+
+touch configure.stamp
```

Comment:
not sure if `configure.stamp` is already part of `.gitignore` by some pattern - but if not, this should get added. Other than that: :rocket:

Reply:
Good point, thanks.